e2e/cypress/support/index.js

```diff
@@ -107,6 +107,11 @@ before(() => {
 
         // * Verify that the server database matches with the DB client and config at "cypress.json"
         cy.apiRequireServerDBToMatch();
+
+        if (Cypress.env('runWithEELicense')) {
+            // * Verify that the server is loaded with license when running tests for EE
+            cy.apiRequireLicense();
+        }
```

Comment:
Here is where the license is being loaded via API, instead of [inserting into the database directly](https://github.com/saturninoabril/mattermost-cypress-docker/compare/es-license?expand=1#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L86-L94).